### PR TITLE
enhancement #11

### PR DESCRIPTION
**Pull request by Arditti93**
#11 Implement a pet class that can be used by the person class

### CODE REVIEW

File: src/pets.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class pets {
```
Comment (SilviaPetruta) [inline src/pets.js:1]: I think the name of the class should be declared with capital P.

**Comment by Arditti93** [comment]
Capital should be fixed now

File: src/pets.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-class pets {
+class Pets {
```
Comment (SilviaPetruta) [inline src/pets.js:1]: Has it also been modified in the main.js file?

Comment (Arditti93) [reply src/pets.js:1]: oops will fix that now

Comment (Arditti93) [reply src/pets.js:1]: should be fixed now

**Pull request merged**